browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -95,6 +96,24 @@ class MarkdownNoteDetail extends React.Component {
     return title
   }
 
+  getPercentageOfCompleteTodo (value) {
+    let splitted = value.split('\n')
+    let numberOfTodo = 0
+    let numberOfCompletedTodo = 0
+
+    for (let i = 0; i < splitted.length; i++) {
+      let trimmedLine = splitted[i].trim()
+      if (trimmedLine.match(/^- \[\s|x\] ./)) {
```

Comment:
You need to add `+` and `*` as statements.

### +

![screen shot 2017-05-06 at 08 20 40](https://cloud.githubusercontent.com/assets/11307908/25773609/0a1af0cc-3235-11e7-8de7-8784db2279b1.png)

### *
![screen shot 2017-05-06 at 08 22 05](https://cloud.githubusercontent.com/assets/11307908/25773614/226a22d8-3235-11e7-94dc-00a21a5ae73c.png)